### ISSUE TITLE
elysia doesn't listen

### ISSUE THREAD

**Issue by draylegend** [JeanMeche/ssr-v19#7]
### Preparations

`bun i`

### Building

After building for production using elysia config `ng b -c elysia` the server doesn't listen.

Using this

```ts
import { AngularAppEngine, createRequestHandler } from '@angular/ssr';
import { Elysia } from 'elysia'

export function app() {
  const server = new Elysia();
  const angularAppEngine = new AngularAppEngine();

  server.get('/*', async (c) => {
    const res = await angularAppEngine.handle(c.request, { server: 'elysia' });
    return res || undefined;
  });

  return server;
}

const server = app();

typeof Bun !== 'undefined' && server.listen(4000, ({ url }) => console.log(`🦊 ${url.href}`));

export const reqHandler = createRequestHandler(server.fetch);
```

I'm facing the following errors:

```js
// polifyl.js and main.js
Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "". Strict MIME type checking is enforced for module scripts per HTML spec.
```

It would be much simpler to develop APIs using one server (angular, elysia & eden) only.

@JeanMeche please help me to understand how to solve these issues, thanks in advance!

**Comment by JeanMeche** [comment]
This is the config I would expect to work : 

```
import { AngularAppEngine, createRequestHandler } from '@angular/ssr';
import { isMainModule } from '@angular/ssr/node';
import { Elysia } from 'elysia';
import { staticPlugin } from '@elysiajs/static';
import { dirname, resolve } from 'node:path';
import { fileURLToPath } from 'node:url';

export function app() {
  const server = new Elysia();
  const angularAppEngine = new AngularAppEngine();

  const serverDistFolder = dirname(fileURLToPath(import.meta.url));
  const browserDistFolder = resolve(serverDistFolder, '../browser');

  server.use(staticPlugin({
    prefix: '',
    assets: browserDistFolder,
    alwaysStatic: true,
  }));

  server.get('/*', async (c) => {
    const res = await angularAppEngine.handle(c.request, { server: 'elysia' });
    return res || undefined;
  });

  console.warn('Elysia server started');

  return server;
}

const server = app();
if (isMainModule(import.meta.url)) {
  const port = process.env['PORT'] || 4000;

  server.listen(port, () => {
    console.log(`Elysia server listening on http://localhost:${port}`);
  });
}

export const reqHandler = createRequestHandler(server.fetch);
```

Especially, 
```
  server.use(staticPlugin({
    prefix: '',
    assets: browserDistFolder,
    alwaysStatic: true,
  }));
```

But it doesn't, it looks like the static plugin doesn't support a having assets in another directory. In our cases `browser`, is a sibling directory of `server`. 

I've open elysiajs/elysia-static#42 to understand a bit more what's happening.

**Comment by YeisonHerrer** [comment]
Hi @draylegend,  

Just wanted to chime in and let you know that with the new Elysia release (v1.2.x) launched today, this issue should now be resolved. The `staticPlugin` works fine with assets in a sibling directory like `browser`.  

I’ve been testing this in a large project that I’m currently migrating, and it’s working well for me. I’d recommend updating to the latest version of Elysia and trying the configuration suggested by @JeanMeche  again.  

Hope this helps. Good luck with your project! 🚀

**Comment by JeanMeche** [comment]
@YeisonHerrer can confirm ! 

fixed by f2f2bcc.
